esi_leap/manager/service.py

```diff
@@ -70,13 +70,14 @@ def _fulfill_leases(self):
                     lease.fulfill(self._context)
                 except Exception as e:
                     LOG.info("Error fulfilling lease: %s", e)
-                    lease.status = statuses.ERROR
+                    lease.status = statuses.WAIT
```

Comment:
If the reason the lease failed to be fulfilled/expired is due to the node being in the wrong state, would we want to distinguish that failure mode (by setting status = WAIT) from other possible failure modes (in which we should set status = ERROR)?

Reply:
Fair question! The answer right now is I think I'd prefer to be in WAIT as much as possible (and thus eligible for a retry), as any error in the lease handling has to be remediated somehow.